srcs/raycast.c

```diff
@@ -0,0 +1,170 @@
+#include "cub.h"
+
+void	initialize_map_pos(t_cub *cub)
```

Comment:
why is this function called initialize_map_pos? Looks more like start_time or something

Reply:
It's a leftover from when I was still setting the proj_plane and dir vectors myself. I moved this now to the init_cub function!